dataactvalidator/scripts/load_cfda.py

```diff
@@ -1,54 +1,89 @@
-from ftplib import FTP
+import os
+from datetime import datetime, timedelta
 from io import BytesIO
 from dataactcore.config import CONFIG_BROKER
 from dataactcore.logging import configure_logging
+import logging
 
 import boto3
+from botocore.handlers import disable_signing
+from botocore.exceptions import ClientError
+
+s3 = boto3.client('s3', )
+logger = logging.getLogger(__name__)
+
+CFDA_FILE_FORMAT = os.path.join('Assistance Listings', 'usaspendinggov', '%Y', '%m-%b',
```

Comment:
Same question as below for the `gsa_bucket`.

Reply:
Makes sense. I can work with Ops to figure out the best way of adding these setting.